components/divider.tsx

```diff
@@ -0,0 +1,33 @@
+/** @jsxImportSource theme-ui */
+import { ThemeUIStyleObject } from 'theme-ui'
+import { Row, Column } from '@carbonplan/components'
+
+interface DividerProps {
+  color?: string
+  sx?: ThemeUIStyleObject
+}
+
+export function Divider({ color = 'gray', sx = {} }: DividerProps) {
+  return (
+    <hr
+      sx={{
+        width: '100%',
+        borderTop: '1px solid',
+        borderColor: color,
+        ...sx,
+      }}
+    />
+  )
+}
+
+export const SectionDivider: React.FC<{ color: string }> = ({ color }) => {
+  return (
+    <>
+      <Row columns={[12]}>
```

Comment:
I think these need to be adjusted to handle the mobile/tablet grid (only 6/8 columns):
![CleanShot 2024-10-03 at 17 18 37@2x](https://github.com/user-attachments/assets/1f11aace-adc0-4781-917a-e9a69ca003c9)